Scripts/script-DemistoLinkIncidents.yml

```diff
@@ -0,0 +1,47 @@
+commonfields:
+  id: DemistoLinkIncidents
+  version: -1
+name: DemistoLinkIncidents
+script: |-
+  var body = {
+      id: args.masterID,
+      otherIncidentsIDs: args.otherIDs.split(','),
+      removeLink: (args.unlink === 'yes')
+  };
+
+  var res = executeCommand('demisto-api-post', {uri: '/incident/links', body: body});
+
+  if (isError(res[0])) {
+      throw res[0].Contents;
+  }
+
+  var response = res[0].Contents.response;
+  var md = tableToMarkdown('Demisto link incidents', response);
+
+  return {
+      ContentsFormat: formats.json,
+      Type: entryTypes.note,
+      Contents: response,
+      HumanReadable: md
+  };
+type: javascript
+tags:
+- DemistoAPI
+comment: Link two or more incidents
+enabled: true
+args:
+- name: masterID
+  required: true
+  default: true
+  description: Incident ID to link incidents with
```

Comment:
Incident ID used for linking other incidents
or 
Incident ID used for linking to other incidents